src/number/format-decimal.js

```diff
@@ -14,18 +14,24 @@ module.exports = function(value, localeData, options) {
 	// round to desired precision
 	value = Math.abs(Math.round(value * precisionScaling) / precisionScaling);
 
-	var integerValue = value | 0;
+	var integerValue = Math.trunc( value );
```

Comment:
I think you'll need the polyfill for this (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc)